modules/data.land/R/landiq2std.R

```diff
@@ -0,0 +1,124 @@
+#' Convert a LandIQ Shapefile into a Standardized Format consisting of
+#' a GeoPackage file to store geospatial information and an associated
+#' CSV file with attributes
+#'
+#' This function reads a LandIQ crop map shapefile downloaded from
+#' https://www.landiq.com/land-use-mapping and processes the data into
```

Comment:
If using DWR nomenclature, canonical URL is probably https://data.cnra.ca.gov/dataset/statewide-crop-mapping

Reply:
Thanks for pointing that out!
```suggestion
#' https://data.cnra.ca.gov/dataset/statewide-crop-mapping and processes the data into
```